.github/workflows/scheduler.yml

```diff
@@ -0,0 +1,67 @@
+name: Test Scheduler
+
+on:
+  workflow_dispatch:
+  schedule:
+   - cron:  '0 12-19/2 * * 1-5'
+permissions:
+  contents: read
+  packages: write
+  deployments: write
+  id-token: write
+  actions: write
+
+env:
+  OWNER: ${{ github.repository_owner }}
+  REPO: ${{ github.event.repository.name }}
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  test_web:
+    name: Test Web Cypress
+    uses: ./.github/workflows/reusable-web-e2e.yml
```

Comment:
@AliaksandrRyzhou, Have these workflows been built in retries for the cypress part? The assumption is that Cypress will remain unstable so we would like the workflow to retry 1-2 times without manual action.

Reply:
It's a good idea, I'll take care about it